ge0/CMakeLists.txt

```diff
@@ -10,5 +10,6 @@ set(SRC
 )
 
 omim_add_library(${PROJECT_NAME} ${SRC})
+target_link_libraries(${PROJECT_NAME} PRIVATE base)
```

Comment:
Looks like it also depends on the coding library. Interesting, why only base lib gave errors?

Reply:
`base` is required to bring transitive dependencies like `utf8cpp`. Absence of other dependencies does not cause errors because `ge0` is a static library.